log/log_example_test.go

```diff
@@ -100,32 +99,6 @@ func ExampleFatal() {
 
 // TODO: Panic
 
-// This example uses command-line flags to demonstrate various outputs
-// depending on the chosen log level.
-func Example_LevelFlag() {
-	setup()
-	debug := flag.Bool("debug", false, "sets log level to debug")
-
-	flag.Parse()
-
-	// Default level for this example is info, unless debug flag is present
-	zerolog.SetGlobalLevel(zerolog.InfoLevel)
-	if *debug {
-		zerolog.SetGlobalLevel(zerolog.DebugLevel)
-	}
-
-	log.Debug().Msg("This message appears only when log level set to Debug")
-	log.Info().Msg("This message appears when log level set to Debug or Info")
-
-	if e := log.Debug(); e.Enabled() {
-		// Compute log output only if enabled.
-		value := "bar"
-		e.Str("foo", value).Msg("some debug message")
-	}
-
-	// Output: {"level":"info","time":1199811905,"message":"This message appears when log level set to Debug or Info"}
-}
```

Comment:
I tried removing the underscore to make `go vet` happy, but I couldn't get this to show up in `godoc`. I think this is because there is no `LevelFlag` function in the `log` package.

Reply:
Just call it `Example`.